assets/js/blocks/cart-checkout/mini-cart/component-frontend.tsx

```diff
@@ -28,7 +28,7 @@ const renderMiniCartFrontend = () => {
 	renderFrontend( {
 		selector: '.wc-block-mini-cart',
 		Block: MiniCartBlock,
-		getProps: ( el: HTMLElement ) => {
+		getProps: ( el ) => {
```

Comment:
Any reason you've removed `HTMLElement` type from `el`?

Reply:
Yes! Now with generics, Typescript can correctly understand what type it is.
It is better not to specify types when possible